src/cdk/table/table.spec.ts

```diff
@@ -655,6 +681,23 @@ class SimpleCdkTableApp {
   template: `
     <cdk-table [dataSource]="dataSource">
       <ng-container cdkColumnDef="column_a">
+        <cdk-header-cell *cdkHeaderCellDef></cdk-header-cell>
+        <cdk-cell *cdkCellDef="let data"> {{data}} </cdk-cell>
+      </ng-container>
+
+      <cdk-header-row *cdkHeaderRowDef="['column_a']"></cdk-header-row>
+      <cdk-row *cdkRowDef="let row; columns: ['column_a']"></cdk-row>
+    </cdk-table>
+  `
+})
+class EvenSimplerCdkTableApp {
```

Comment:
`TableWithBooleanDataSource`?

("EvenSimpler" doesn't tell you anything really)